_layouts/partners/state-and-local.html

```diff
@@ -12,6 +12,9 @@
         <h1 class="partners-title">{{ page.title }}</h1>
         <div class="tablet:grid-col-12 partners-border"></div>
         <div class="tablet:grid-col-12 partners-subtitle">{{ page.body | markdownify }}</div>
+        <div class="tablet:grid-col-12 partners-body-wide">
```

Comment:
A couple thoughts on the approach of assigning width (which existed already):

- If we want this to match the subtitle above, could we just reuse the same class, or rename / create a new class which is used for both? At least for the styles which are concerned with width.
- I'm not sure it makes sense to use the grid classes if we proceed to override the width anyways. That being said, I'd prefer if we used the default grid layout, but it'd probably require a bit of refactoring to create separate rows and resize based on viewport.

e.g.

```html
<div class="grid-row">
  <div class="grid-col-12 tablet:grid-col-10 partners-subtitle">
    {{ page.body | markdownify }}
  </div>
</div>
<div class="grid-row">
  <div class="grid-col-12 tablet:grid-col-10">
    {{ page.disclaimer | markdownify }}
  </div>
</div>
```

Reply:
For the first note - it's not supposed to match the subtitle. The font is supposed to match the smaller one used for `partners-body` that's how I got in this mess 😅. 

With regard to the second point. Totally agree - I took out the grid reference to reduce the redundancy. BUT a refactor is clearly in order. I'm not sure why `partners-body` is setting width at all when we'd ideally be leaning on the grid to do that for us. I don't want to start cleaning that up right now though. 

I can try to dig into that tomorrow morning.